osf/models/base.py

```diff
@@ -516,6 +516,9 @@ def load(cls, guid, select_for_update=False):
         except cls.MultipleObjectsReturned:
             return None
 
+    def get_guid(self):
+        return self.versioned_guids.first().guid
+
```

Comment:
Remove after https://github.com/CenterForOpenScience/osf.io/pull/10820 is merged.